lib/rspec/legacy_formatters.rb

```diff
@@ -2,6 +2,7 @@
 require "rspec/legacy_formatters/version"
 require "rspec/legacy_formatters/adaptor"
 require 'stringio'
+require 'rspec/core'
```

Comment:
Thanks for pointing that out. :scream:   I will rebase and try without this.  We're using Teamcity 7 to run continuous integration tests, and for some reason I had been using a fork with this change.  Let me clean it up and try again.